src/core/Form/StatusText/StatusText.test.tsx

```diff
@@ -43,13 +38,6 @@ describe('props', () => {
       expect(container.firstChild).toHaveClass('fi-status-text--error');
     });
   });
-
-  describe('disabled', () => {
-    it('is null when disabled', () => {
```

Comment:
Same here, maybe we should check that the aria live is not enabled when a component is disabled?

Reply:
This is now tested.

Reply:
I'm wondering how we should handle this. It might be safer to keep aria-live attribute in DOM always when status text component is used. When there is no content, live mode should be off and same goes with disabled. Possible values are off, polite or assertive. This test should also check that the attribute exists rather than just checking there is children present.

Reply:
When there's no content, I don't see why it should be off. The way I think about it is that having an empty region with assertive aria-live mode means that it is ready to read out loud whatever content it gets. Turning it off while waiting for content doesn't really accomplish anything in my opinion. And for the test, do you mean we should add a test to check if an empty element has the aria-live attribute as well?

Reply:
Added tests to see that the element has an aria-live attribute even when empty and that the mode is 'off' when the element is disabled.